### ISSUE TITLE
Table title

### ISSUE THREAD

**Issue by calpirro** [gitana/alpaca#652]
Hi,

in a field of type = array, like this.

`
"attachment": {
   "type": "array",
   "title": "Attachments",
   "items": {
      "type": "object",
      "properties": {
              etc...
      }
   }
}
`
The **title** of the table (Attachments in thi case) is not shown using Alpaca version 1.5.24. It works fine using previous versions (I tried 1.5.23 and 1.5.22 and it's OK)

**Comment by calpirro** [comment]
The same happens in version 1.5.27
What is worse is that if you have the array collapsed ("collapsed": true, in schema options) it is impossible to add / edit / remove elements because you doun't have to "Label" to click and open the array...

**Comment by calpirro** [comment]
I've done some testing, version 1.5.27
I took the example number 2 in the official documentation of table field [http://www.alpacajs.org/docs/fields/table.html](url) and added a Label in options and is not shown, then if you add a  **"collapsed": true** in the options you can't see the table and can't click in the Label to show it. If you remove the "type": "table" then works fine (but show as an aray, not as a table)

Here is the full code:

`$("#field2").alpaca({
    "data": [{
        "key": "key1",
        "title": "title1",
        "amount": 2.53,
        "rating": "good"
    }, {
        "key": "key2",
        "title": "title2",
        "amount": 1.80,
        "rating": "poor"
    }, {
        "key": "key3",
        "title": "title3",
        "amount": 5.60,
        "rating": "excellent"
    }],
    "schema": {
        "type": "array",
        "items": {
            "type": "object",
            "properties": {
                "key": {
                    "type": "string",
                    "title": "Key"
                },
                "title": {
                    "type": "string",
                    "title": "Title"
                },
                "amount": {
                    "type": "number",
                    "title": "Amount"
                },
                "approved": {
                    "type": "boolean",
                    "title": "Approved"
                },
                "rating": {
                    "type": "string",
                    "enum": ["poor", "good", "excellent"],
                    "title": "Rating"
                }
            }
        }
    },
    "options": {
        "label": "LABEL",
        "type": "table",
        "collapsed": true,
        "collapsible": true,
        "items": {
            "fields": {
                "rating": {
                    "type": "select"
                }
            }
        },
        "form": {
            "buttons": {
                "submit": {
                    "click": function() {
                        var value = this.getValue();
                        alert(JSON.stringify(value, null, "  "));
                    }
                }
            }
        }
    }
});`

**Comment by ajboni** [comment]
@calpirro  did you ever fixed it?

**Comment by calpirro** [comment]
No... I avoid use this options...BTW is this project alive? Lasts release was in May, 2019....
